extension.js

```diff
@@ -109,7 +109,9 @@ class Extension {
     }
 
     _destroySubMenu() {
-        this._bootOptionsSubMenu.destroy();
-        this._bootOptionsSubMenu = null;
+        if (this._bootOptionsSubMenu) {
```

Comment:
Does this fix #5? Or is it simply a "null" check for safety?

Reply:
I added that because if something failed on enable (on my gentoo box here, bootctl list failed), the extension will throw an error on disable, I didn't check #5

Reply:
Ah, makes sense. #5 might continue to persist then.